integ-test/src/test/java/org/opensearch/sql/legacy/AggregationExpressionIT.java

```diff
@@ -121,6 +121,7 @@ public void hasGroupKeyMaxAddLiteralShouldPass() {
         rows("f", 1));
   }
 
+  @Ignore("Handled by v2 engine which returns 'name': 'Log(MAX(age) + MIN(age))' instead")
```

Comment:
Do we need to keep this test?

Reply:
I can find similar test with math function on aggregation but not exactly the same. I added it to comparison test just in case. Thanks for reminding!